dev/create-release/release-tag.sh

```diff
@@ -122,6 +122,12 @@ if ! is_dry_run; then
   git push origin $RELEASE_TAG
   if [[ $RELEASE_VERSION != *"preview"* ]]; then
     git push origin HEAD:$GIT_BRANCH
```

Comment:
Hm, so to clarify, we set `set -e` on the top, meaning that the script will fail immediately if any command fails with non-zero exit.

I assume that there's a case when `git push origin $RELEASE_TAG` and `git push origin HEAD:$GIT_BRANCH` are successfully executed but the tag still doesn't exist? If that's the case LGTM.

Reply:
When `git push origin HEAD:$GIT_BRANCH` doesn't succeed, the PR proposes to call out the exact error instead of failing silently. 
In addition, the reason why we have to `| grep origin` is that, `git branch -r --contains tags/_d_tag` exits 0 even if `git push origin HEAD:$GIT_BRANCH` doesn't execute.